docs/Getting Started.md

```diff
@@ -52,7 +52,6 @@ services:
       - SQLITE=true
       - HWID=RANDOM
       - LOGLEVEL=INFO
-      - LOGFILE=/dev/stdout
```

Comment:
If we have issues with log to stdout we should provide a default value of a writable path.

Reply:
I would not recommend that, as we already have historical problems with our current `/home/py-kms` path stuff...

Reply:
Is that because we run rootless?

Reply:
Additionally the default value is already `STDOUT` - but of the process itself, see `start.py` - can we resolve this or do you still think we should set that?

Reply:
> Is that because we run rootless?

Yes. And because of the bug you have mentioned inside moby.